girder_tech_journal/__init__.py

```diff
@@ -120,11 +160,11 @@ def checkSubmission(self, filterParams, submission, searchObj, targetVal, catego
                 searchObj,
                 targetVal)
         # Capture all revisions under each object
-        revisionInfo = list(self.model('folder')
-                                .childFolders(parentType='folder',
-                                              parent=submission,
-                                              user=self.getCurrentUser()
-                                              ))
+        revisionInfo = list(ModelImporter.model('folder')
```

Comment:
We don't want to use `ModelImporter` anymore unless it's absolutely necessary (which it typically isn't). We should instead (for this example):
```
from girder.models.folder import Folder

Folder().childFolders(
```

Reply:
You should be able to use some find -> replace macros to change the majority of these.